bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -362,4 +362,14 @@ default void visit(Consumer<ComponentExporter> callback) throws Exception {}
     default String getParentPagePath() {
         return null;
     }
+
+    /**
+     * Returns the form container's json.
+     * 
+     * @return A String that consists of the form container json
+     */
+    @JsonIgnore
+    default String getFormDefinition() {
```

Comment:
This is already part of FormStructureParser and it is merged. Why do we have another PR for this ?